spec/factories/approval_requests.rb

```diff
@@ -0,0 +1,8 @@
+FactoryBot.define do
+  factory :approval_request do
+    approval_request_ref { "MyString" }
+    workflow_ref { "MyString" }
+    state { "MyString" }
```

Comment:
Might want this to match either a default ( if we add one ) or one of the `enum` states.

Reply:
Update to :undecided